src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int BOUND = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int number = new Random().nextInt(BOUND);

```

Comment:
this should not be a field, only BOUND should. This defeats the purpose of the class - Lottery won't return different balls, this value only will be set once